### PR TITLE
Renewal status cleanup

### PR DESCRIPTION
**Pull request by xendk**
Cleanup as mentioned in http://platform.dandigbib.org/issues/761#note-25

Openruth not updated.

### CODE REVIEW

**Comment by xendk** [comment]
Scrutinizer flags an issue that was fixed the Drupal CS a year ago: https://www.drupal.org/node/2710067

File: modules/fbs/includes/fbs.loan.inc [hunk 1]
```diff
@@ -86,10 +86,19 @@ function fbs_loan_renew($account, $loans) {
     // Using an array for renewalStatus is an odd choice, but we'll only
     // consider the loan renewed if 'renewed' is the only status.
     if ($loan->renewalStatus == array('renewed')) {
-      $result[$loan->loanDetails->loanId] = DingProviderLoan::STATUS_RENEWED;
+      $result[$loan->loanDetails->loanId] =
```
Comment (JrgenGNielsen) [inline modules/fbs/includes/fbs.loan.inc]: Scrutinizer don't like this

Comment (xendk) [inline modules/fbs/includes/fbs.loan.inc]: As I wrote, this is a false positive that's been fixed in Coder a year ago.

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Pull request merged**